pandas/core/arrays/datetimes.py

```diff
@@ -170,6 +171,8 @@ class DatetimeArrayMixin(dtl.DatetimeLikeArrayMixin):
     # Constructors
 
     _attributes = ["freq", "tz"]
+    _freq = None
+    _tz = None
```

Comment:
These will be needed later, are orthogonal to the rest of this PR.

Reply:
Sure.  Though I'd like to collect things like this in a Get It Out Of The Way PR before long